### DISCUSSION TITLE
Cannot find bluesky.createSession in version 0.18.x

### DISCUSSION THREAD

**Discussion by CPULL** [myConsciousness/atproto.dart#1840] [General]
Hello, I started just yesterday to use Flutter and Dart, to build a simple Bluesky application.
I started using the version 0.15.0 that was recommended by a tutorial, and (after some work) I was able to do some activity (read a list, add/remove items, do a post)

I needed then the ability to add an InputField in the application, and I had very hard time because of some dependencies that should have been upgraded.
At the end I moved to V0.18.5 for Bluesky (and 0.13.2 for atproto) but with this version I cannot find anymore the `bsky.createSession()` call.
Are the APIs changed between 0.15 and 0.18? Or I am doing some rookie mistake?

**Comment by myConsciousness** [comment]
Hi @CPULL ,

we already talked about this on bluesky chat, could you fix it?

**Comment by CPULL** [comment]
Hello,
yes, you told me how to do it on Bluesky DMs.
Yes, it works perfectly.

**Reply by heroheman** [reply]
How did you solve it? I have the same issue right now.

**Reply by CPULL** [reply]
```
import 'package:atproto/atproto.dart' as atp;
import 'package:bluesky/bluesky.dart' as bsky;

final session = await atp.createSession(
      service: 'bsky.social',
      identifier: userHandle,
      password: userAppPassword,
);
final bluesky = bsky.Bluesky.fromSession(session.data);
```

**Reply by heroheman** [reply]
Thank you! The docs should be updated with this.
